demo/features/taa-txn-author-acceptance.feature

```diff
@@ -0,0 +1,92 @@
+
+      Examples:
+         | Acme_capabilities                      | Schema_name    |
+         | --taa-accept                           | driverslicense |
+         | --taa-accept --multitenant             | driverslicense |
+         | --taa-accept --revocation              | driverslicense |
+
+   @T001a-TAA @taa_required
+   Scenario Outline: accept the ledger TAA and write to the ledger via endorser
+      Given we have "2" agents
+         | name  | role     | capabilities        |
+         | Acme  | endorser | <Acme_capabilities> |
+         | Bob   | author   | <Bob_capabilities>  |
+      And "Acme" connects to a ledger that requires acceptance of the TAA
+      And "Bob" connects to a ledger that requires acceptance of the TAA
+      And "Acme" and "Bob" have an existing connection
+      When "Acme" accepts the TAA
+      And "Bob" accepts the TAA
+      And "Acme" has a DID with role "ENDORSER"
+      And "Acme" connection has job role "TRANSACTION_ENDORSER"
+      And "Bob" connection has job role "TRANSACTION_AUTHOR"
+      And "Bob" connection sets endorser info
+      And "Bob" has a DID with role "AUTHOR"
+      And "Bob" authors a schema transaction with <Schema_name>
+      And "Bob" requests endorsement for the transaction
+      And "Acme" endorses the transaction
+      Then "Bob" can write the transaction to the ledger
+      And "Bob" has written the schema <Schema_name> to the ledger
+
+      Examples:
+         | Acme_capabilities          | Bob_capabilities           | Schema_name    |
+         | --taa-accept               | --taa-accept               | driverslicense |
+         | --taa-accept --multitenant | --taa-accept --multitenant | driverslicense |
+
+   @T002-TAA @taa_required
+   Scenario Outline: Revoke credential using a ledger with TAA required
+      Given we have "2" agents
+         | name  | role     | capabilities        |
+         | Faber | verifier | <Acme_capabilities> |
+         | Bob   | prover   | <Bob_capabilities>  |
+      And "Faber" connects to a ledger that requires acceptance of the TAA
+      And "Faber" accepts the TAA
+      And "Faber" and "Bob" have an existing connection
+      And "Bob" has an issued <Schema_name> credential <Credential_data> from "<issuer>"
+      And "Faber" revokes the credential
+      And "Faber" successfully revoked the credential
+      When "Faber" sends a request for proof presentation <Proof_request> to "Bob"
+      Then "Faber" has the proof verification fail
+
+      Examples:
+         | issuer | Acme_capabilities                        | Bob_capabilities | Schema_name       | Credential_data   | Proof_request     |
+         | Faber  | --taa-accept --revocation --public-did   |                  | driverslicense_v2 | Data_DL_MaxValues | DL_age_over_19_v2 |
+
+   @T003-TAA @taa_required
+   Scenario Outline: Fail to publish revoked credential using a ledger with TAA required, and fix the ledger
+      Given we have "2" agents
```

Comment:
This is the scenario that we ran into with LSBC - a failed ledger transaction corrupted the revocation registry and caused future revocations to fail.  The "posts a correction to the ledger" step invokes the new aca-py endpoint to fix the ledger.